src/BuildScriptGenerator.Common/LoggingConstants.cs

```diff
@@ -13,6 +13,7 @@ public static class LoggingConstants
         public const string ApplicationInsightsInstrumentationKeyEnvironmentVariableName
             = "ORYX_AI_INSTRUMENTATION_KEY";
 
+        public const string ApplicationInsightsConnectionStringKeyEnvironmentVariableName = "ORYX_AI_CONNECTION_STRING";
```

Comment:
nit: let's follow the same pattern with these longer variables/strings and have the value be on a new line:

```cs
public const string ApplicationInsightsInstrumentationKeyEnvironmentVariableName
    = "ORYX_AI_INSTRUMENTATION_KEY";

public const string ApplicationInsightsConnectionStringKeyEnvironmentVariableName
    = "ORYX_AI_CONNECTION_STRING";

public const string OryxDisableTelemetryEnvironmentVariableName
    = "ORYX_DISABLE_TELEMETRY";
```